spec/models/event_spec.rb

```diff
@@ -11,6 +11,21 @@
     it { is_expected.to validate_presence_of :start_date }
     it { is_expected.to validate_presence_of :end_date }
     it { is_expected.to validate_presence_of :main_email_contact }
+
+    context 'with start date before end date' do
+      let(:event) { FactoryGirl.build :event, start_date: DateTime.new(2016, 5, 20).in_time_zone, end_date: DateTime.new(2016, 5, 21).in_time_zone }
+      it { expect(event.valid?).to be_truthy }
```

Comment:
Did you run `rubocop` locally in your fork? I'm not sure if it accepts this data creation. The snap is failing due to ruby version, so it is not giving the correct feedback right now. =(